### ISSUE TITLE
why was the batchsize  1.

### ISSUE THREAD

**Issue by Lzc6996** [D-X-Y/caffe-faster-rcnn#27]
I find that in this code, the batchsize is always 1, how can i use a big batchsize without using iter_size?

**Comment by D-X-Y** [comment]
Because the different image has different size, thus for the first layer, the input tensor should need some padding for alignment. batchsize = 1 will avoid this padding.

**Comment by Lzc6996** [comment]
Thanks for you answer.
I will try to modify it, because batchsize=1 make batchnorm doesn't work which is very important for me.

**Comment by D-X-Y** [comment]
If you are using the pre-trained model, you can fix the parameters of the BatchNorm. If not, it's a problem.
